helm_deploy/cmd-api/templates/cronjob-shift-changes.yaml

```diff
@@ -20,6 +20,6 @@ spec:
             image: quay.io/ukhomeofficedigital/openjdk11
             command: ["/bin/sh", "-c"]
             args:
-              - 'JWT=$(curl -X POST {{ .Values.env.OAUTH_ROOT_URL }}/oauth/token?grant_type=client_credentials -H "Content-Type: application/json" -H "Content-Length: 0" -H "Authorization: Basic $(echo -n {{ .Values.secrets.API_CLIENT_ID }}:{{ .Values.secrets.API_CLIENT_SECRET }} | base64)"); TOKEN=$(echo -n $JWT | grep -o -E ''[-a-zA-Z0-9\._]{100,}''); curl http://cmd-api.{{ .Values.cron.namespace }}.svc.cluster.local/notifications/refresh -H "Authorization: Bearer $(echo -n $TOKEN)";'
+              - 'JWT=$(curl -X POST {{ .Values.env.OAUTH_ROOT_URL }}/oauth/token?grant_type=client_credentials -H "Content-Type: application/json" -H "Content-Length: 0" -H "Authorization: Basic $(echo -n ''{{ .Values.secrets.API_CLIENT_ID }}:{{ .Values.secrets.API_CLIENT_SECRET }}'' | base64)"); TOKEN=$(echo -n $JWT | grep -o -E ''[-a-zA-Z0-9\._]{100,}''); curl http://cmd-api.{{ .Values.cron.namespace }}.svc.cluster.local/notifications/refresh -H "Authorization: Bearer $(echo -n $TOKEN)";'
           restartPolicy: Never
```

Comment:
The preprod secret has characters in it that need quoting, the dev one didn't by coincidence